finalPrep/modZeroHero.js

```diff
@@ -55,17 +99,51 @@ var excuse = 'I think I forgot to lock up my 1992 Toyota Coralla. Be right back.
 //   - citiesDestroyed (array)
 //   - luckyNumbers (array)
 //   - address (object with following key/values: number , street , state, zip)
-
+var scaryMonster = {
+  name: "Sleepless Toddler",
+  smell: "poopy",
+  weight: 23,
+  citiesDestroyed: ["Denver", "Pueblo", "Greeley"],
+  luckynumbers: [6, 24, 19],
+  address: {
+    number: 8327,
+    street: "Ames Way",
+    state: "Colorado",
+    zip: "80003",
+  }
+}
 
 // Create a new class called SuperHero
 // - Your class should have the following DYNAMIC values
-//   - name 
+//   - name
 //   - superpower
-//   - age 
+//   - age
 // - Your class should have the following STATIC values
 //   - archNemesis, assigned to "The Syntax Error"
 //   - powerLevel = 100
-//   - energyLevel = 50 
+//   - energyLevel = 50
+class SuperHero {
+  constructor(name, superpower, age){
+    this.name = name;
+    this.power = superpower;
+    this.age = age;
+    this.archNemesis = "The Syntax Error";
+    this.powerLevel = 100;
+    this.energyLevel = 50;
+  }
+  sayName() {
+    this.name;
+  }
+  maximizeEnergy() {
+    this.powerLevel = 1000;
+  }
+  gainPower (number) {
+    this.powerLevel = (number + this.energyLevel);
```

Comment:
Here, the instructions ask you to:

> create a method called gainPower, it should take an argument of a number and INCREASE the _powerLevel_ by that number.

I expected to see the following:
```
gainPower(number) {
    this.powerLevel = this.powerLevel + number
}
```
There wasn't any mention in the instructions about including the `energyLevel` instance variable.